src/resources/Resource.js

```diff
@@ -58,7 +60,16 @@ export default class Resource {
   }
 
   async parseResponse (response) {
-    const data = await response.json()
+    let text = await response.text()
```

Comment:
Read the next few lines... In case of an incorrect response (non-JSON) I want to know what that response was. With `.json()` there is no way of knowing what the original response was.